lib/filter_param.cpp

```diff
@@ -864,3 +968,87 @@ vector<double> FilterParam::init_stable_coef(const double a0, const double a) co
 	return coef;
 }
 
+/* # フィルタ構造体
+ *   振幅特性図の描画
+ * 	 leftとrightで描画範囲の指定[0:0.5]  
+ *
+ * # 引数
+ * vector<double> &coef : 係数列
+ * string &filename : 出力するファイル名
+ * double left : 帯域の左端正規化周波数 [0:0.5)
+ * double right : 帯域の右端正規化周波数 (0:0.5]
+ * 
+ */
+void FilterParam::gprint_amp
+(const vector<double> &coef, const string &filename, const double left, const double right) const
+{
+	BandParam range(BandType::Pass, left, right);
+	FilterParam fparam(n_order, m_order, range, 250, 0, 5.0);
+	auto freq_res = fparam.freq_res(coef);
+
+	double grange = (right-left)*2*M_PI/250; //固定値の事前計算
```

Comment:
```suggestion
	constexpr double grange = (right-left)*2.0*M_PI/250.0; //固定値の事前計算
```
left, rightは入れれないかもしれません。
コンパイル時定数にならないかもなので

`constexpr`を使わない場合、計算できる部分は事前に処理するべきです
2.0*PI/250 ---> PI / 125